pyomo/contrib/appsi/solvers/tests/test_persistent_solvers.py

```diff
@@ -1359,7 +1390,8 @@ def test_param_updates(self, name: str, opt_class: Type[PersistentSolver]):
         m.obj = pe.Objective(expr=m.y)
         m.c1 = pe.Constraint(expr=(0, m.y - m.a1 * m.x - m.b1, None))
         m.c2 = pe.Constraint(expr=(None, -m.y + m.a2 * m.x + m.b2, 0))
-        m.dual = pe.Suffix(direction=pe.Suffix.IMPORT)
+        if not opt_class is MAiNGO:
```

Comment:
This is fine for now. We have an open backlog item to redesign the solver tests from scratch, so we can try to find a more clever way to do this in the future.

One request, though - universally change the check to:
```
if opt_class != MAiNGO
```
(Assuming that doesn't error for some Pyomo-related reason...)